src/symmetry.jl

```diff
@@ -68,8 +68,8 @@ Filter out the symmetry operations that respect the symmetries of the discrete B
 function symmetries_preserving_kgrid(symmetries, kcoords)
     kcoords_normalized = normalize_kpoint_coordinate.(kcoords)
     T = eltype(kcoords[1])
-    tol = T <: Rational ? 0 : sqrt(eps(T))
-    is_approx_in(x, X, tol) = any(y -> norm(x-y) ≤ tol, X)
+    atol = T <: Rational ? 0 : sqrt(eps(T))
+    is_approx_in(x, X, tol) = any(y -> isapprox(x, y; atol), X)
```

Comment:
No need to specify `atol`. What you are doing here is the default behaviour.

Reply:
hm indeed!

Reply:
although, no: default is to use rtol, which will not be approximate when comparing to zero

Reply:
ok, let's keep it as is then.